build/Targets/References.targets

```diff
@@ -120,10 +84,7 @@
     <When Condition="'$(IncludeCPSReferences)' == 'true'">
       <ItemGroup>
 
-        <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\src\Dependencies\CPS\CPS.csproj">
-          <Project>{03ab838c-1b9d-4eca-856f-476616c35f0c}</Project>
-          <Name>CPS</Name>
-        </ProjectReference>
+        <PackageReference Include="Microsoft.VisualStudio.ProjectSystem.SDK" Version="15.0.737" PrivateAssets="None" />
```

Comment:
No, we should do the opposite and get rid of the nuspec. We currently have a checked in nuget.exe and call pack on that - instead we can simply call msbuild /t:Pack on these library projects to produce the package. Then this version will be the source of truth for the packages as well.